applications/MedApplication/custom_io/med_model_part_io.cpp

```diff
@@ -362,6 +362,9 @@ class MedModelPartIO::MedFileHandler
         // read only by default, unless other settings are specified
         med_access_mode open_mode;
 
+        // Fix to allow windows conversion from whatever eldritch format is using to something convertible to c_str()
+        std::string med_file_mame{rFileName.string()};
```

Comment:
mame? 😄

Reply:
a f**k, now I see it now. I`ll Fix it later :P